cli/src/build-settings.ts

```diff
@@ -13,35 +13,46 @@ function isStringDict(x: JSON.Object): x is Dict<string | null> {
 
 export default class BuildSettings {
   private rustc: string;
+  private nodeVersion: string;
   private env: Dict<string | null>;
 
-  constructor(rustc: string, env: Dict<string | null>) {
+  constructor(rustc: string, nodeVersion: string, env: Dict<string | null>) {
```

Comment:
Forgot to mention in the review: the type of this parameter should also by `string | null`.